Minesweeper.py

```diff
@@ -43,68 +28,86 @@
                 print(board[y][x], end=" ")
         print()
 
-    # Print the game stats
+def print_stats():
     print("Moves:", moves)
     print("Flags:", flags)
     print("Mines found:", mines_found)
```

Comment:
The print statements in `print_stats()` are not aligned properly with the function definition. ```diff -      print("Moves:", moves) +    print("Moves:", moves) -      print("Flags:", flags) +    print("Flags:", flags) -      print("Mines found:", mines_found) +    print("Mines found:", mines_found) ``` ---